demos/_includes/DemoActionsUtil.php

```diff
@@ -6,9 +6,40 @@
 
 use Atk4\Data\Model;
 use Atk4\Data\Model\UserAction;
+use Atk4\Data\Persistence\Array_;
 use Atk4\Ui\Exception;
 use Atk4\Ui\Form;
 
+/**
+ *  @property string $age    @Atk4\Field()
+ *  @property string $city   @Atk4\Field()
+ *  @property string $gender @Atk4\Field()
+ */
+class ArgModel extends Model
```

Comment:
our testing/demos Model should be used here to test if field aliasing is used properly